code/src/components/ItemList.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react'
+import { useSelector } from 'react-redux'
+import { Item } from 'components/Item'
+import { useDispatch } from 'react-redux'
+import { todos } from 'reducers/todos'
+
+export const ItemList = () => {
+    const dispatch = useDispatch()
+    const items = useSelector((state) => state.todos.items)
+    const [category, setCategory] = useState('')
+
+    const handleSubmit = (event) => {
```

Comment:
Hmm, there is no option to submit this form, so I cannot set filters to view only one category items

Reply:
I didn´ t manage to fix that the hole way, and I don't want to take it away because I want to try to solve it later on

Reply:
👍